src/common/low_precision_transformations/tests/fake_quantize_transformation.cpp

```diff
@@ -183,13 +183,6 @@ const std::vector<FakeQuantizeTransformationTestValues> fakeQuantizeTransformati
      ngraph::element::i8,
      {{ngraph::element::f32, {{}, {-128.f}, {0.01f}}}, {ngraph::element::f16, {{}, {-128.f}, {0.01f}}}},
      true},
-    // dot interval
-    {LayerTransformation::createParamsI8I8(),
-     {256ul, {}, {0.f}, {2.55f}, {2.55f}, {2.55f}},
-     {256ul, {}, {0.f}, {2.55f}, {1.f}, {1.f}},
-     ngraph::element::Type_t::i8,
-     {{ngraph::element::f32, {{}, {}, {2.55f}}}},
-     true},
 
```

Comment:
dot interval FQ is being const-folded in this PR, thus the old compareFunction test case is removed